### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by bishopconnor59**
I had a hard time finding any significant problems with my paper. I spent a lot of time fixing tag problems and spelling mistakes where I could find them.

### CODE REVIEW

**Comment by whanley** [comment]
Looks good so far

File: 1905-08-03.xml [hunk 1]
```diff
@@ -2607,5399 +2612,5360 @@
...
-                            “toujours servi de parc qua- rantenaire et que le Gouvernement n'a pas
-                            d’autre emplacement disponible pour servir au débarquement du bétail en
-                            qnestion.’’ A ce'a la Municipalité ne trouve rien à répondre et se borne
-                            à émettre le vœu que l’affectation do Chatby au débarquement ne soit que
-                            provisoire.</p>
-                        <p>C’est pour elle le fruit de mille efforts perdu. 11 ne lui aura donc rien
-                            servi de débarrasser Cbatby de tous les établissements insalubres,
-                            abattoir, parc quarantanaire, lazaret, tanneries, pour les transporter
-                            au Mex et ruiner cette localité. Les espérances d’habiles spéculateurs
-                            fondées sur une plus-value énorme des terrains se trouvera du mê
```
Comment (whanley) [inline 1905-08-03.xml]: date

File: 1905-08-03.xml [hunk 2]
```diff
@@ -1112,10 +1113,10 @@
                             Karachi, and Bombay accelerated service about <date when="1905-08-18"
                                 >August 18</date>. To Aden, Karachi, Bombay, Colombo, Madras,
                             Rangoon, and Calcutta about <date when="1905-07-20">August
-                                18</date>.</p>
+                            18</date>.</p>
```
Comment (whanley) [inline 1905-08-03.xml]: the when attribute in this and surrounding dates is wrong

**Comment by whanley** [comment]
Generally looks good. Use the `feature=` attribute where appropriate. Pages 7, 8 and 9 have some problems with hierarchy of divs. Look at these pointers: https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/#1-get-the-page-hierarchy-right

Don't forget to make the above changes to your revision!

**Comment by whanley** [comment]
The page hierarchy is not quite right. Look at line 8279--you close the page 8 div before you include the last items in that page. Look at these pointers: https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/#1-get-the-page-hierarchy-right

**Comment by huoyanzhizhou** [comment]
Just like Dr. Hanley said, the structure in the outline window is not correct. Please check your outline window to correct it. Also, remember to use the feature "local" at page 3.

**Comment by whanley** [comment]
Thanks, Connor. We'll call this complete.

**Pull request merged**